src/appkit.cs

```diff
@@ -23671,4 +23676,10 @@ interface NSOpenGLLayer
 		void Draw (NSOpenGLContext context, NSOpenGLPixelFormat pixelFormat, double t, ref CVTimeStamp ts);
 	}
 
+	[Protocol (IsInformal=true)]
```

Comment:
Maybe we can mark the method as required so that it can be autocompleted?

Technically it's not correct, but then again technically we should bind categories using `[Category]` and not `[Protocol (IsInformal=true)]`, but that doesn't provide autocompletion either.